src/pages/SetPasswordPage.js

```diff
@@ -96,7 +97,9 @@ class SetPasswordPage extends Component {
     render() {
         const buttonText = this.props.translate('setPasswordPage.setPassword');
         const sessionError = this.props.session.errors && ErrorUtils.getLatestErrorMessage(this.props.session);
-        const error = sessionError || ErrorUtils.getLatestErrorMessage(this.props.account);
+        const accountError = this.props.account.errors && ErrorUtils.getLatestErrorMessage(this.props.account);
```

Comment:
Could we move the `this.props.account.errors` check into the ErrorUtils function?

Reply:
Same for the `sessionError` too

Reply:
We can check if the onyxData.errors is empty and return early with empty string. How does that sound?

Reply:
If you mean that we could do that within `ErrorUtils.getLatestErrorMessage`, I agree.

Reply:
Updated!